src/reusable-components/Error.tsx

```diff
@@ -6,8 +6,11 @@ type Props = {
 }
 
 export default function Error({ error, wrapperClassName }: Props) {
-  const routeError = useRouteError();
-  const errorToShow = routeError ? routeError as Error : error;
+  const routeError = useRouteError() as Error;
+  /*SG Would be nice to use nullish coalescing operator if you check for the existence of a variable to use it after
+  Nullish coalescing checks if variable is undefined / null
+  */
+  const errorToShow = routeError ?? error;
```

Comment:
I'll replace any ternary operator with the nullish coalescing operator where possible, thanks.

Reply:
If you want to check if it's the same behavior: https://github.com/DanutGavrus/1-app-for-a-Live-Chat-using-React-TS-Firebase-and-Tailwind/pull/19/commits/08ae4b8a0f5efc323ec538bf60050d4baba6ecc6